features/nationaliteit.feature

```diff
@@ -11,19 +11,21 @@ Functionaliteit: Bepalen van de actuele nationaliteit van een persoon
 
     @gba
     Scenario: de persoon heeft één actuele nationaliteit
-      Gegeven de persoon met burgerservicenummer '000009830' heeft de volgende gegevens
-      | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
-      | 1      | 4         | 0001                  | 001                   |                          |                                    |                 | 19750707                        |
+      Gegeven de persoon met burgerservicenummer '000009830' heeft een 'nationaliteit' met de volgende gegevens
+      | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+      | 0001                  | 001                   | 19750707                        |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                              |
-      | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 000009830                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '000009830' een nationaliteit met de volgende gegevens
-      | naam                  | waarde   |
-      | nationaliteit         | 0001     |
-      | redenOpname           | 001      |
-      | datumIngangGeldigheid | 19750707 |
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000009830                       |
+      | fields              | nationaliteiten                 |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
+      | naam                              | waarde      |
+      | nationaliteit.code                | 0001        |
```

Comment:
als het goed werkt krijg je ook nationaliteit.omschrijving en redenOpname.omschrijving

Reply:
> als het goed werkt krijg je ook nationaliteit.omschrijving en redenOpname.omschrijving. Of wordt hier alleen gecheckt dat ten minste deze gegevens in het antwoord zitten (niet falen wanneer er ook nog andere gegevens in zitten)?

Ik heb nu niet in detail gekeken naar de correctheid van de scenario's. Dat laat ik graag aan over aan de rest van het team. Ook het toevoegen van de nodige aanpassingen in de scenario's. Verder zou ik er niet voor kiezen om alleen te valideren wat in de dan stappen is opgegeven. Want dan moeten er andere dan stappen worden bedacht om te valideren dat je niet meer krijgt dan gevraagd

Reply:
ok, ik probeer het te snappen. Kan ik concluderen dat de door Frank genoemde gegevens zowel aan de given als aan de dan stap kan worden toegevoegd?